src/bin/main.rs

```diff
@@ -52,7 +52,10 @@ impl SecureWalletFile for WalletFile {
 async fn main() -> anyhow::Result<()> {
     if let Err(err) = exec().await {
         // display the error message (if any)
-        eprintln!("{}", err);
+        match err.downcast_ref::<requestty::ErrorKind>() {
+            Some(requestty::ErrorKind::Interrupted) => {}
```

Comment:
if I don't shade it, it automatically display `CTRL+C`.
I decided to shade it, because this `CTRL+C` is just the display of `requestty::ErrorKind::Interrupted`.
This means that we will have a different UX depending on when the SIGTERM is catched

I mean:
- During a prompt wait I will get this display
- During any other phase (eg: while I'm fetching notes) nothing display

So, in order to unify the UX (as this is not the right PR) I decided to write this way.

@ZER0 does it make sense to you too?

Reply:
Yes. But then add a comment there explaining exactly that.
With a `TODO` about intercept the interrupt so that we can provide tot the user a meaningful message (you can also already open an issue about that, and put the issue number in the comment).